dbms/src/Databases/DatabaseCloud.cpp

```diff
@@ -755,6 +755,13 @@ void DatabaseCloud::renameTable(const Context & context, const String & table_na
 }
 
 
+UInt32 DatabaseCloud::getTableMetaModTime(const String & table_name)
+{
+	LOG_INFO(log, "getTableMetaModTime() isn't implemented yet");
```

Comment:
Just return 0 (as dummy implementation), don't log.
Reason: otherwise will be too many messages when selecting from system.tables.